nexus/reconfigurator/planning/src/planner.rs

```diff
@@ -2689,6 +2693,37 @@ pub(crate) mod test {
             .collect::<BTreeSet<_>>()
     }
 
+    #[track_caller]
+    pub(crate) fn assert_sim_planning_makes_no_changes(
+        sim: &mut ReconfiguratorCliTestState,
+    ) {
+        sim.run(&["blueprint-plan latest latest"]).unwrap();
```

Comment:
hmm, I like this approach over the alternative of using a more structured form (e.g. using the Rust clap enums). I've found the latter to get pretty cumbersome esp as new arguments are added.